scripts/liotad.service

```diff
@@ -0,0 +1,11 @@
+#
+# systemd unit configuration
+#
+[Unit]
+Description=Little Internet of Things Agent Daemon
+
+[Service]
+ExecStart=/etc/liota/packages/liotad.py
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I`m not sure but I think better to add a new line in both the files liotad.service & liotad.socket

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline